src/xdocs/config_coding.xml

```diff
@@ -7088,6 +7088,35 @@ class C {
         <source>
 &lt;module name=&quot;SuperFinalize&quot;/&gt;
         </source>
+        <p>Example:</p>
+        <source>
+public class A {
+   @Override
+      protected void finalize() throws Throwable {
+          try{
+            System.out.println("Finalize of sub class");  //release resources, perform cleanup
+            } catch(Throwable t) {
+                throw t;
+            } finally { // OK, calls super.finalize
+                System.out.println("Calling finalize");
+                super.finalize();
+            }
+      }
+}
+class B {
+   @Override
+      protected void finalize() throws Throwable {
+            try{
+                // Statements;
+            } catch() {
+                //body;
+            }
+            finally { // causes violation does not call super.finalize
```

Comment:
Please put comment "//violation" on line where checkstyle put violation.
Prove it CLI execution.

Reply:
Please reply "done" to each review item you addressed and pushed a fix